test/e2e/config/fixture.yaml

```
@@ -32,3 +36,6 @@ intervals:
   default/wait-deployment: ["5m", "1s"]
   default/wait-namespace-deleted: ["20s", "1s"]
   ironic/wait-deployment: ["10m", "2s"]
+  default/wait-provisioning: ["20s", "1s"]
+  default/wait-provisioned: ["5m", "5s"]
+  default/wait-deprovisioning: ["1m", "2s"]
```

We can probably have the same here
```suggestion
  default/wait-deprovisioning: ["1m", "10ms"]
```

Tiny nit: There is no newline at the end of the file (this is why github shows the :no_entry: sign here). It doesn't really matter by causes unnecessary diffs when adding more lines